internal/setup/releaseV1.go

```diff
@@ -0,0 +1,110 @@
+package setup
+
+import (
+	"bytes"
+	"fmt"
+
+	"golang.org/x/crypto/openpgp/packet"
+	"gopkg.in/yaml.v3"
+
+	"github.com/canonical/chisel/internal/pgputil"
+)
+
+func ParseReleaseV1(baseDir, filePath string, data []byte) (*Release, error) {
```

Comment:
nit: same comment as above. Does this need to be exported?

Reply:
You are right!